src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -146,4 +147,43 @@ public void testSaveAndLoadMessages() throws PersistentDataStoreException {
     Assert.assertEquals(contentTwo, resultMessageTwo.getContent());
     Assert.assertEquals(creationTwo, resultMessageTwo.getCreationTime());
   }
+
+  @Test
+  public void testSaveAndLoadHashtags() throws PersistentDataStoreException {
+    UUID idOne = UUID.fromString("10000000-2222-3333-4444-555555555555");
+    UUID idOnwerOne = UUID.fromString("10000000-2222-3333-3333-555555555555");
+    String content1 = "soccer";
+    Instant creationOne = Instant.ofEpochMilli(1000);
+    Boolean createdFromUser1 = false;
+    Hashtag inputHashOne = new Hashtag(idOne, idOnwerOne, content1, creationOne, createdFromUser1);
+
+    UUID idTwo = UUID.fromString("20000000-2222-3333-4444-555555555555");
+    UUID idOnwerTwo = UUID.fromString("20000000-2222-3333-3333-555555555555");
+    String content2 = "football";
+    Instant creationTwo = Instant.ofEpochMilli(1000);
+    Boolean createdFromUser2 = false;
+    Hashtag inputHashTwo = new Hashtag(idTwo, idOnwerTwo, content2, creationTwo, createdFromUser2);
+
+    // save
+    persistentDataStore.writeThrough(inputHashOne);
+    persistentDataStore.writeThrough(inputHashTwo);
+
+    // load
```

Comment:
So is `writeThrough()` basically like a mutator and `loadHashtags()` is like the accessor that retrieves the set tags?

Reply:
Yep, writethrough() lets me write to the database while loadHashtags let me see what it is inside the database.